extensions/default/src/DicomWebDataSource/utils/getImageId.js

```diff
@@ -13,7 +13,7 @@ function buildInstanceWadoUrl(config, instance) {
 
   const paramString = params.join('&');
 
-  return `${config.wadoUriRoot}?${paramString}`;
+  return `${config.baseUrl}/${config.wadoURLPrefix}?${paramString}`;
```

Comment:
Use the function added to the config to get the query parameters/setup, eg make it:
config.getUrl('wado', `?{paramString}`)